python/hail/keytable.py

```diff
@@ -295,8 +295,8 @@ def aggregate_by_key(self, key_expr, agg_expr):
 
         The scope for both ``key_expr`` and ``agg_expr`` is all column names in the input :class:`KeyTable`.
 
-        For more information, see the documentation on writing `expressions <../overview.html#expressions>`_
-        and using the `Hail Expression Language <../reference.html#HailExpressionLanguage>`_.
+        For more information, see the documentation on writing :ref:`expressions <overview-expressions>`
+        and using the `Hail Expression Language <https://hail.is/expr_lang.html>`_
```

Comment:
Once #1411 goes in then the section reference will be `exprlang` in RST. I believe my pull request already fixed this instance. For now, the reference should be `../expr_lang.html`.